generator/src/generators/pascals-triangle.cr

```diff
@@ -8,9 +8,9 @@ class PascalsTriangleGenerator < ExerciseGenerator
 
   def test_cases
     test_cases = [] of JSON::Any
-    JSON.parse(data)["cases"].each do |group|
-      group.each do |a, b|
-        test_cases.concat(b) if a.as_s == "cases"
+    JSON.parse(data)["cases"].as_a.map do |group|
```

Comment:
Same here. Probably could use `each` or some kind of `flat_map` (instead of shovel `<<`).